src/sonic-host-services-data/debian/sonic-host-services-data.featured.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Can't we wait for DB ready and start this daemon?
I don't like timers because it makes SONiC even more asynchronous than it is.

Reply:
I'm following the design steps of Hostcfgd. 

FYI, In the FeatureDaemon Ctor, we are waiting for the CONFIG_DB_INITIALIZED flag, so functionally there wont be any problem. 
```
        self.config_db = ConfigDBConnector()
        self.config_db.connect(wait_for_init=True, retry_on=True)
```

Reply:
Resolving this, if there aren't any other comments